### PR TITLE
Unicode fix

### LINKED ISSUE #1130: UnicodeEncodeError: 'ascii' codec can't encode character u'\xf6' in position 17: ordinal not in rang ...
Happened on Backend

[View in Dump Manager](https://master.tomato-lab.org/dumpmanager/group/Exception__3f56c40650ff925c87e6c7ed374f0537)

Description:
```
{'type': 'UnicodeEncodeError', 'subject': "'ascii' codec can't encode character u'\\xf6' in position 17: ordinal not in range(128)"}
```
Trace:
```
/code/tomato/lib/util.py, line 24, in call
 return fn(*args, **kwargs)

/code/tomato/topology.py, line 290, in timeout_task
 top.sendNotification(subject="Topology timeout warning: %s" % top, message="The topology %s will time out soon. This means that the topology will be first stopped and afterwards destroyed which will result in data loss. If you still want to use this topology, please log in and renew the topology." % top)

/code/tomato/topology.py, line 195, in sendNotification
 permission.user.sendNotification(ref=['topology', self.idStr], **kwargs)

/code/tomato/auth/__init__.py, line 222, in sendNotification
 self._sendMail(subject, message, fromUser)

/code/tomato/auth/__init__.py, line 241, in _sendMail
 mail.send("%s <%s>" % (self.realname or self.name, self.email), subject, message, from_=from_)

/code/tomato/lib/mail.py, line 12, in send
 msg = MIMEText(message)

/usr/lib/pypy/lib-python/2.7/email/mime/text.py, line 30, in __init__
 self.set_payload(_text, _charset)

/usr/lib/pypy/lib-python/2.7/email/message.py, line 226, in set_payload
 self.set_charset(charset)

/usr/lib/pypy/lib-python/2.7/email/message.py, line 262, in set_charset
 self._payload = self._payload.encode(charset.output_charset)
```

### PR DESCRIPTION
**Pull request by t-gerhard**
Closes #1130

### CODE REVIEW

File: backend/tomato/lib/mail.py [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
-	if isinstance(to, list):
-		to = ", ".join(to)
-	msg = MIMEText(message)
-	msg['Subject'] = subject
-	msg['From'] = from_
-	msg['To'] = to
 	try:
+		msg = MIMEText(EMAIL_MESSAGE_TEMPLATE % {'realname': to_realname, 'message': message}, 'plain', 'utf-8')
+		msg['Subject'] = Header(EMAIL_SUBJECT_TEMPLATE % {'subject': subject}, 'iso-8859-1')
+		msg['From'] = "%s <%s>" %(Header(from_realname, 'iso-8859-1'), from_addr) if from_realname and from_addr else EMAIL_FROM
```
Comment (dswd) [inline backend/tomato/lib/mail.py]: Why iso-8859-1 ?

**Comment by t-gerhard** [comment]
Changed to utf-8

**Comment by dswd** [comment]
Did you test that?

**Comment by t-gerhard** [comment]
Of course

**Pull request merged**